src/monopoly/monopoly_player.js

```diff
@@ -75,13 +71,15 @@ MonopolyPlayer.prototype.owesRent = function(property) {
 };
 
 MonopolyPlayer.prototype.owns = function(property) {
-  return property.owner === this;
+  return property.owner && property.owner.id === this.id;
```

Comment:
We should make sure to be consistent with the style we use to check this whether we check for object equivalence or id equivalence.. I'm not necessarily partial to one way or the other but we should definitely be consistent